src/index.js

```diff
@@ -76,17 +79,43 @@ function validateFormData(data) {
 /*
  *    GENERATION
  */
-
 function generateTextInputAndLabel(formField) {
-  const textInputElement = document.createElement("input");
-  textInputElement.type = formField.type;
-  textInputElement.id = formField.id;
+  if (formField.multiline !== true) {
+    const textInputElement = document.createElement("input");
+    textInputElement.type = formField.type;
+    textInputElement.id = formField.id;
+
+    const textInputLabelElement = document.createElement("label");
+    textInputLabelElement.htmlFor = formField.id;
+    textInputLabelElement.innerText = formField.label;
+
+    return [textInputElement, textInputLabelElement];
+  } else {
+    const textInputElement = document.createElement("textarea");
+    textInputElement.id = formField.id;
+
+    const textInputLabelElement = document.createElement("label");
+    textInputLabelElement.htmlFor = formField.id;
+    textInputLabelElement.innerText = formField.label;
+
+    return [textInputElement, textInputLabelElement];
+  }
```

Comment:
This can be simplified to something like : 

```
    const elementToCreate = formField.multiline ? 'textarea' : 'input';
    const textInputElement = document.createElement(elementToCreate);
    if (elementToCreate = 'input') {
        textInputElement.type = formField.type;
    }
    textInputElement.id = formField.id;

    const textInputLabelElement = document.createElement("label");
    textInputLabelElement.htmlFor = formField.id;
    textInputLabelElement.innerText = formField.label;

    return [textInputElement, textInputLabelElement];
```